### DISCUSSION TITLE
Content is not displayed when inside a Frame in .net 8

### DISCUSSION THREAD

**Discussion by Wbenay** [dotnet/maui#20534] [General]
The same problem is related this discussion : [](https://github.com/dotnet/maui/issues/19274), but is not resolved. 

https://github.com/dotnet/maui/issues/19274

![image](https://github.com/dotnet/maui/assets/38923341/ea0db01e-a674-49da-9e60-39f963ebd625)

Exemple other project:
![image](https://github.com/dotnet/maui/assets/38923341/e058a10a-cf91-42a2-9c65-f1051c30fead)

 <Frame Margin="5" 
CornerRadius="10" 
BackgroundColor="White" 
BorderColor="#d5d5d5" 
HasShadow="False">

     <StackLayout>
         <StackLayout Orientation="Horizontal" 
              Spacing="10" 
              Padding="{OnPlatform Android='5', iOS='10,50,10,20'}">

             <Border WidthRequest="100"
         HeightRequest="100"
         StrokeThickness="10"
         StrokeShape="RoundRectangle 100" >

                 <Border.Shadow>
                     <Shadow Brush="Transparent" 
                     Offset="5,5" 
                     Opacity=".3" />
                 </Border.Shadow>
                 <Image Grid.Row="0" 
                HorizontalOptions="Center" 
                WidthRequest="90" 
                HeightRequest="90" 
                Source = "{Binding ImagemPessoa}">
                 </Image>
             </Border>
             <Label Text="{Binding Nome}" 
            VerticalOptions="Center"
            FontFamily="{StaticResource LatoRegular}"
            FontAttributes="Bold"
            FontSize="20"
            TextColor="#343434"/>
         </StackLayout>

         <StackLayout Padding="5">
             <Label FontSize="Medium" TextColor="#343434">
                 <Label.FormattedText>
                     <FormattedString>
                         <FormattedString.Spans>
                             <Span Text="Empresa:" FontAttributes="Bold" FontSize="Medium" />
                             <Span Text=" " />
                             <Span Text="{Binding Empresa}" />
                         </FormattedString.Spans>
                     </FormattedString>
                 </Label.FormattedText>
             </Label>

             <Label FontSize="Medium" TextColor="#343434">
                 <Label.FormattedText>
                     <FormattedString>
                         <FormattedString.Spans>
                             <Span Text="Departamento:" FontAttributes="Bold" FontSize="Medium" />
                             <Span Text=" " />
                             <Span Text="{Binding Departamento}" />
                         </FormattedString.Spans>
                     </FormattedString>
                 </Label.FormattedText>
             </Label>
         </StackLayout>
     </StackLayout>
 </Frame>

**Comment by marius-jantea** [comment]
Hello! Did you find a way to make this work?
My assumption is that your issue comes from the StrokeShape that is being set on the border. I also opened a discussion where I gave more details [here](https://github.com/dotnet/maui/discussions/21356)

**Reply by Wbenay** [reply]
Hi, I didn't find a solution, so I removed Border! 
I'll try your solution! Thanks.

**Reply by marius-jantea** [reply]
I re-created an issue for MAUI. You cand find it here:
https://github.com/dotnet/maui/issues/21390
I also gave the details on why this is happending when setting a stroke shape to the border, and why it is happening only when the border is nested in a Compatibility / Legacy layout type. 
I hope it will be fixed.
